org_chart.js

```diff
@@ -1,73 +1,132 @@
-let elon = {
-  name: "Elon Musk",
-  email: "elon.musk@musky.com",
-  position: "Boss",
+"use strict";
+
+function Employee(name, email, position, gender, project){
+  this.name = name;
+  this.email = email;
+  this.position = position;
+  this.gender = gender;
+  if (!project) {
+  this.project = "";
+  } else {
+  this.project = project;
+  }
 }
 
-let new_hires = [
-  {
-    name: "Alan Turing",
-    email: "alan.turing@musky.com",
-    position: "Computer Scientist",
-    project: "Cryptography"
-  },
-  {
-    name: "Grace Hopper",
-    email: "grace.hopper@musky.com",
-    position: "Computer Scientist",
-    project: "Compilers"
-  },
-  {
-    name: "Donald Knuth",
-    email: "donald.knuth@musky.com",
-    position: "Mathematician",
-    project: "Algorithm Analysis"
+Employee.prototype.addSkill = function(skills) {
+  if (!this.skills) {
+    this.skills = [];
```

Comment:
This check and initialisation can be eliminated if you just initialise `skills` in the constructor